tests/integration/test_dataset.py

```diff
@@ -48,9 +40,6 @@ def test_dataset(client, rand_gen):
     assert dataset.description == description
 
     dataset.delete()
```

Comment:
It would be good to add a check for the deleted dataset here by querying for the dataset
```
with pytest.raises(SomeException):
   client.get_dataset(dataset.uid)
```

Reply:
This already exists on line 44.